resources/user-agents/browsers/firefox-on-kaios/firefox-kaios-1-0.json

```diff
@@ -0,0 +1,35 @@
+{
+  "division": "Firefox #MAJORVER#.#MINORVER# for KaiOS 1.0",
+  "versions": [37],
+  "sortIndex": 3167,
+  "lite": false,
+  "standard": false,
+  "userAgents": [
+    {
+      "userAgent": "Firefox #MAJORVER#.#MINORVER# for KaiOS 1.0",
+      "browser": "firefox",
+      "platform": "KaiOS_1_0",
+      "engine": "Gecko_Dynamic",
+      "device": "general Mobile Phone",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Firefox #MAJORVER#.#MINORVER# for KaiOS 1.0",
+        "Version": "#MAJORVER#.#MINORVER#"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (Mobile; #DEVICE#; rv:#MAJORVER#.#MINORVER#*)*Gecko/#MAJORVER#.#MINORVER#*Firefox/#MAJORVER#.#MINORVER#*KaiOS/1.0*",
```

Comment:
Why do you not use the platform placeholder?

Reply:
I was planning to initially, but then realized that there isn't a case (yet) where this version of firefox would be on a different version of the OS.

I can replace though.